docs/api/sql/Raster-operators.md

```diff
@@ -963,6 +963,47 @@ Output:
 -3.000000
 ```
 
+### RS_SetValues
+
+Introduction: Returns raster with new values for specified pixels for designated band.
```

Comment:
Please clearly explain how this setValue works. It should cover the region specified by the min point from colX and rowY.